storage_controller/src/service.rs

```diff
@@ -5724,7 +5730,11 @@ impl Service {
             let locked = self.inner.read().unwrap();
             if let Some(node) = locked.nodes.get(&register_req.node_id) {
                 if node.registration_match(&register_req) {
```

Comment:
nit: can you add a comment to `registration_match` that says we are intentionally skipping over the https port and that this is a transient state?